gwctl/pkg/cmd/get/get.go

```diff
@@ -79,6 +80,17 @@ func runGet(args []string, params *utils.CmdParams, flags *getFlags) {
 		}
 		gwPrinter.Print(resourceModel)
 
+	case "gatewayclass", "gatewayclasses":
+		filter := resourcediscovery.Filter{Namespace: ns}
+		if len(args) > 1 {
+			filter.Name = args[1]
+		}
+		resourceModel, err := discoverer.DiscoverResourcesForGatewayClass(filter)
+		if err != nil {
+			panic(err)
```

Comment:
FYI: Please do checkout comment https://github.com/kubernetes-sigs/gateway-api/pull/2836#discussion_r1513893310 about not panic-ing in newly added code.